impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -653,6 +650,18 @@ GaussianBlurPipeline::FragmentShader::KernelSamples GenerateBlurInfo(
     x_offset = 1;
   }
 
+  // This is a safe-guard to make sure we don't overflow the fragment shader.
+  // The kernel size is multiplied by 2 since we'll use the lerp hack on the
+  // result. In practice this isn't throwing away much data since the blur radii
+  // are around 53 before the down-sampling and max sigma of 500 kick in.
+  //
+  // TODO(https://github.com/flutter/flutter/issues/150462): Come up with a more
+  // wholistic remedy for this.  A proper downsample size should not make this
```

Comment:
holistic ?

Reply:
Both spellings are accepted.  After a slight rabbit hole I can report that it isn't some sort of british-ism.  "Wholistic" came about about 100 years ago for some goofy reason, probably to sell snake oil.